packages/rocketchat-livechat/.app/imports/client/visitor.js

```diff
@@ -38,6 +38,11 @@ export default {
 		return this.token.get();
 	},
 
+	getDepartment() {
+		const data = this.getData();
+		return data ? data.department : null; 
```

Comment:
we can use ```

Reply:
@ggazzo !
Ternary condition removed.
Thx.